internal/version/stats.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package version
+
+import "hash/crc32"
+
+func (v *Info) Stats(cb func(name string, val float64)) {
```

Comment:
Use `reportValue` or similar. When reading the code `cb` confused me.

Reply:
callback? but okay yeah